### PR TITLE
OCM-10628 | ci: enable NAME_PREFIX for rosacli CI jobs

### PR DESCRIPTION
**Pull request by xueli181114**

### CODE REVIEW

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@ tests:
   steps:
     cluster_profile: aws-sd-qe
     env:
+      NAME_PREFIX: pr-rosa-hcp
```
Comment (yasun1) [inline ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml:47]: We do not need to name the different prefixes for each presubmit job; just the same one, e.g., `pr-` is enough.

Comment (xueli181114) [reply ci-operator/config/openshift/rosa/openshift-rosa-master__e2e-presubmits.yaml]: Updated

**Comment by xueli181114** [comment]
/pj-rehearse pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp-advanced

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml [hunk 2]
```diff
@@ -26,6 +26,7 @@ tests:
     env:
       CHANNEL_GROUP: stable
       IMPORTANCE: (Critical,High)
+      NAME_PREFIX: rosa-sts-f3
```
Comment (yasun1) [inline ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml:29]: As same as the presubmit jobs, do we need different names for the periodic jobs?

**Comment by xueli181114** [comment]
```
lixue@Xue-Lis-MacBook-Pro rosa % rosa describe cluster -c 2dcvp4tgt1o72aq0olamlt8pbqof9v2q

Name:                       pr-rosacli-slkq
Domain Prefix:              pr-rosacli-slkq
Display Name:               pr-rosacli-slkq
ID:                         2dcvp4tgt1o72aq0olamlt8pbqof9v2q
External ID:                99a17004-838e-46ee-bc37-6a91bbba29f6
Control Plane:              ROSA Service Hosted
OpenShift Version:          4.16.7
Channel Group:              stable
DNS:                        pr-rosacli-slkq.67n0.s3.devshift.org
AWS Account:                301721915996
AWS Billing Account:        301721915996
API URL:                    https://api.pr-rosacli-slkq.67n0.s3.devshift.org:443
Console URL:                
Region:                     us-west-2
Availability:
 - Control Plane:           MultiAZ
 - Data Plane:              MultiAZ

Nodes:
 - Compute (Autoscaled):    3-6
 - Compute (current):       1
Network:
 - Type:                    OVNKubernetes
 - Service CIDR:            172.31.0.0/24
 - Machine CIDR:            10.0.0.0/16
 - Pod CIDR:                192.168.0.0/18
 - Host Prefix:             /25
 - Subnets:                 subnet-0204b3849cb33cd35, subnet-07564ef207f047c26, subnet-07056dfe27788bd4c, subnet-055fdb6d10e9950ac, subnet-06b7de67e334cd775, subnet-025083a597aff9971
Proxy:
 - HTTPProxy:               http://10.0.0.216:8080
 - HTTPSProxy:              https://10.0.0.216:8080
 - NoProxy:                 quay.io
Additional trust bundle:    REDACTED
EC2 Metadata Http Tokens:   required
Role (STS) ARN:             arn:aws:iam::301721915996:role/pr-rosacli-slkq-HCP-ROSA-Installer-Role
Support Role ARN:           arn:aws:iam::301721915996:role/pr-rosacli-slkq-HCP-ROSA-Support-Role
Instance IAM Roles:
 - Worker:                  arn:aws:iam::301721915996:role/pr-rosacli-slkq-HCP-ROSA-Worker-Role
Operator IAM Roles:
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-kube-system-control-plane-operator
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-kube-system-kms-provider
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-openshift-cloud-network-config-controller-cloud-
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-openshift-image-registry-installer-cloud-credent
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-openshift-ingress-operator-cloud-credentials
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-openshift-cluster-csi-drivers-ebs-cloud-credenti
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-kube-system-kube-controller-manager
 - arn:aws:iam::301721915996:role/pr-rosacli-slkq-kube-system-capa-controller-manager
Managed Policies:           Yes
State:                      ready 
Private:                    No
Delete Protection:          Disabled
Created:                    Aug 26 2024 09:27:50 UTC
User Workload Monitoring:   Disabled
Details Page:               https://qaprodauth.console.redhat.com/openshift/details/s/2lBrma0KsBeZNSrevUhJMnwqQmH
OIDC Endpoint URL:          https://oidc.os1.devshift.org/2dcvmaukr6rmlbuhiioai9ffsdmn95di (Managed)
Audit Log Forwarding:       Enabled
External Authentication:    Disabled
Audit Log Role ARN:         arn:aws:iam::301721915996:role/pr-rosacli-slkq
Failed Inflight Checks:
	ID:                 4f1ef4e5-830c-41b3-a00f-fbf1d9f32b1d
	Last run:           Aug 26 2024 09:31:24 UTC
	Invalid configurations on subnet 'subnet-0204b3849cb33cd35' have been identified: 
		Details for 'Egress URL access issues':
			- https://cdn03.quay.io:443 (SSL certificate problem: unable to get local issuer certificate)
	Invalid configurations on subnet 'subnet-07056dfe27788bd4c' have been identified: 
		Details for 'Egress URL access issues':
			- https://infogw.api.openshift.com:443 (SSL certificate problem: unable to get local issuer certificate)
	Invalid configurations on subnet 'subnet-07564ef207f047c26' have been identified: 
		Details for 'Egress URL access issues':
			- https://quayio-production-s3.s3.amazonaws.com:443 (SSL certificate problem: unable to get local issuer certificate)

	Please run `rosa verify network -c 2dcvp4tgt1o72aq0olamlt8pbqof9v2q` after adjusting the cluster's network configuration to remove the warning
lixue@Xue-Lis-MacBook-Pro rosa % 
```

File: ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml (hunk 2 above)
Comment (xueli181114) [reply ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml:29]: I prefer to have f3/f7 in the name prefix. So I can know if the leak is caused by critical+high jobs or medium+low jobs and the leak frequency.

**Pull request merged**